packages/react-native-renderer/src/ReactNativeFiberInspector.js

```diff
@@ -223,10 +223,11 @@ function getInspectorDataForViewAtPoint(
           }
 
           closestInstance =
```

Comment:
This was previously using a private property on `ReactFabricHostComponent`, right? This seems to be the only place we need to access `internalInstanceHandle`.

Reply:
> This was previously using a private property on ReactFabricHostComponent, right?

Yeah.

> This seems to be the only place we need to access internalInstanceHandle.

There's another one in `ReactFabricComponentTree`. What are you suggesting?

Reply:
Just trying to see if we can avoid the prop and re-use the existing one, but since this is now on canonical, it probably doesn't matter as much.

Reply:
We could use symbol properties on the public instance to avoid creating new properties and keep encapsulation, but I want to make public instances lazy and that would prevent me from doing it.